### PR TITLE
Financial instruments rw neo4j

### PR DESCRIPTION
**Pull request by danielcopaciu**

### CODE REVIEW

**Comment by Nesze** [comment]
👍

**Comment by tamas-molnar** [comment]
lgtm

File: financial-instruments-rw-neo4j-blue@.service [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Change killmode from "control-group" to "none" to let Docker remove work correctly.
+KillMode=none
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill "$(docker ps -q --filter=name=%p-%i_)" > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm "$(docker ps -q --filter=name=%p-%i_)" > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker history coco/financial-instruments-rw-neo4j:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/financial-instruments-rw-neo4j:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export APP_PORT=8080; \
+  export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
+  /usr/bin/docker run --rm --name %p-%i_$(uuidgen) -p $APP_PORT --env "NEO_URL=http://$HOSTNAME:8080/__neo4j-blue/db/data" --env "APP_PORT=$APP_PORT" --env "BATCH_SIZE=600" --env "GRAPHITE_ADDRESS=graphite.ft.com:2003" --env "GRAPHITE_PREFIX=coco.services.$ENV.financial-instruments-rw-neo4j-blue.%i" coco/financial-instruments-rw-neo4j:$DOCKER_APP_VERSION;'
```
Comment (BerniVarga) [inline financial-instruments-rw-neo4j-blue@.service]: The run command could be a bit better formatted/aligned - each of the env variables in a new line - for better readability. :) (same applies for the blue rw)

**Comment by kovacshuni** [comment]
+1

**Comment by BerniVarga** [comment]
+1

**Comment by Nesze** [comment]
Tested in XP.

**Comment by ancabusuioc** [comment]
Tested in pre-prod

**Pull request merged**